lib/components/Layout/styles/layout.css

```diff
@@ -5,6 +5,17 @@
   left: 0;
 }
 
+.container, .container a, .container p {
+  font-family: var(--fontPrimary);
```

Comment:
Can we use `--vds-font-family-body` instead of `--fontPrimary`? I'm in the process of removing the latter as a dependency on virtru-design-params.

Reply:
Yep I meant to do that. Thanks!